build/llhttp.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const { join, resolve } = require('path')
+const { copyFileSync, rmSync } = require('fs')
+const { execSync } = require('child_process')
+const TMP = require('os').tmpdir()
+
+const REPO = 'git@github.com:dnlup/llhttp.git'
+const CHECKOUT = 'undici_wasm'
+const REPO_PATH = join(TMP, 'llhttp')
+const WASM_OUT = resolve(__dirname, '../lib/llhttp')
+
+let code = 0
+
+try {
+  execSync(`git clone ${REPO}`, { stdio: 'inherit', cwd: TMP })
+  execSync(`git checkout ${CHECKOUT}`, { stdio: 'inherit', cwd: REPO_PATH })
+  execSync('npm ci', { stdio: 'inherit', cwd: REPO_PATH })
```

Comment:
What is `npm ci`?

Reply:
Ref: https://docs.npmjs.com/cli/v7/commands/npm-ci

It performs a clean installation by removing `node_modules` and using the lockfile, which makes it faster too. `llhttp` has a lockfile, that is the reason I used it.

Reply:
Maybe add a comment?